eos-autoupdater/main.c

```diff
@@ -19,7 +19,7 @@
 
 #include <eos-updater/dbus.h>
 #include <eos-updater/resources.h>
-#include <libeos-updater-util/config.h>
+#include <libeos-updater-util/config-util.h>
```

Comment:
Done

Reply:
Looks like you renamed the `.c` files in `libeos-updater-util/` but not the ones in `libeos-updater-util/tests/` which I think by convention are supposed to have the same names as the files they're testing. To avoid another round of back and forth, I'll assume that was an oversight and do it for you.